File: trunk/src/VS/server/org.mobileapi.server.windows.shared/FileLoader.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Linq;
using System.Net;
using System.Text;
using System.Threading.Tasks;

namespace org.mobileapi.server.windows.shared
{
    public class FileLoader
    {
        public static string ServerPath;

        public string Load(String filename)
        {
            string path = ServerPath + filename;
            StreamReader reader = File.OpenText(path);
            string responseFromServer = reader.ReadToEnd();
            Console.WriteLine(responseFromServer);
            reader.Close();
            return responseFromServer;
        }
    }
}

File: trunk/src/Java/org.mobileapi.server.api/src/org/mobileapi/helper/FileLoader.java
package org.mobileapi.helper;

public class FileLoader {

    public static String ServerPath;

    public String Load(String filename)
    {
        String path = ServerPath + filename;
        
        /*
        StreamReader reader = File.OpenText(path);
        string responseFromServer = reader.ReadToEnd();
        Console.WriteLine(responseFromServer);
        reader.Close();
        return responseFromServer;
        */
        
        return "";
    }
}
